### PR TITLE
Improve sponsoring and merge changes

### PR DESCRIPTION
**Pull request by cpaelzer**
Along reviewing and sponsoring MRs I've came by a few weakly defined areas which I wanted to improve. In particular sponsoring and -v on merge uploads.

### CODE REVIEW

File: PackageMerging.md [hunk 1]
```diff
@@ -862,7 +862,25 @@ The switches are:
 * `-sa` = include orig tarball (required on a merge)
 * `-vXYZ` = include changelog since XYZ
 
-Changes should be from the last Ubuntu version.
+Since our merge upload represents all changes that happened in Debian since
+our last merge plus anything added as part of the merge itself `-v` should
+usually point to the last published Ubuntu version. Here an example:
```
Comment (s-makin) [inline PackageMerging.md:867]:
```suggestion
usually point to the last published Ubuntu version. Here's an example:
```

File: PackageMerging.md [hunk 2]
```diff
@@ -862,7 +862,25 @@ The switches are:
...
+* Then Ubuntu had a fix in `1.3-1ubuntu2`
+* But in the meantime Debian merged upstream as `1.4-1`
+* And then added a fix in `1.4-2`
+* New Ubuntu will be `1.4-2ubuntu1`
+* `-v` should here be set to `1.3-1ubuntu2`
+  * Thereby the .changes file will include `1.4-1`, `1.4-2`, and `1.4-2ubuntu1`
+  * That represents all the changes that happened from the perspective of a Ubuntu user that will upgrade from `1.3-1ubuntu2 to `1.4-2ubuntu1`
+
+If sponsoring a merge or any other upload for someone else, please remember the
+need to sign their upload with your key. See[Sponsor a package](Sponsorship.md#sponsor-a-package)
```
Comment (s-makin) [inline PackageMerging.md:879]:
```suggestion
need to sign their upload with your key. See [Sponsor a package](Sponsorship.md#sponsor-a-package)
```

**Review by s-makin — COMMENTED** [review]
Generally LGTM, just a couple nits

File: Sponsorship.md [hunk 3]
```diff
@@ -69,3 +69,18 @@ You should also keep good notes of who has
...
 work on. These notes will be helpful both for finding sponsors in the future,
 and as endorsers on your future applications for upload rights.
+
+## Sponsor a package
+
+This is not very different than uploading your own, but after ensuring that
+the upload follows all quality standards you want to sign their content
+with your key.
+
+To do so you need to tools (`dpkg-buildpackage` or `debsign`) know about that.
```
[the remark this answers is not in the document]
Comment (cpaelzer) [reply Sponsorship.md]: Thank you, I've rewritten this whole section - WDYT now @s-makin ?

**Review by s-makin — APPROVED** [review]
LGTM :)

File: Sponsorship.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (s-makin) [reply Sponsorship.md]: Much better, thanks!

[the remark this answers is not in the document]
Comment (cpaelzer) [reply Sponsorship.md]: Thank you!

**Pull request merged**